### ISSUE TITLE
Add option to hide "system indices"

### ISSUE THREAD

**Issue by curry684** [ElasticHQ/elasticsearch-HQ#429]
Kibana hides system indices by default, with names starting with a dot. I'd love to have the same option in ElasticHQ, as right now most overview screens require several clicks before showing the useful data, as index lists show 10 items by default and those are in many systems the least interesting ones.

<img width="960" alt="elastichq" src="https://user-images.githubusercontent.com/1455673/44975290-5efb8d80-af62-11e8-9050-c756a751b8a9.png">

In the screenshot above all the interesting information is on page 3, the actual indices.

Similarly an option to change the default page size would be awesome as I'd set it to 25 right away, more than enough space for that.

**Comment by royrusso** [comment]
Fair point. I can add that to the settings section.

**Comment by curry684** [comment]
Hey @royrusso , thanks for implementing the filter, that's awesome, but this skips my second remark about the page size. Any chance of that too?   :)

**Comment by royrusso** [comment]
Good call. Can you create a separate issue for that? This way we can have the UI guys look at it.

**Comment by royrusso** [comment]
@curry684 I think the table size is a dupe in the current release: #436

**Comment by curry684** [comment]
it is, added my 👍 there, thanks again for the awesome tool, we run it as the main frontend to our cluster and only use Kibana when diving into data  🥂
